configure-oidc.md

```diff
@@ -75,3 +75,128 @@ When issuing a `cosign sign` command, add the flags to pass in the client secret
 ```
 cosign sign -y --fulcio-url=$FULCIO_URL --rekor-url=$REKOR_URL --oidc-issuer=$OIDC_ISSUER_URL $IMAGE --oidc-client-secret-file=<my secret filename> --oidc-client-id=<my client ID>
 ```
+
+## Github (federated via Keycloak)
+
+### Obtain Client ID and Secret from Github OAuth App
+
+To integrate RHTAS with Github as an identity provider, you must register your Github OAuth App in your Keycloak realm. 
+
+Locate your Github Oauth App, or create a new Github Oauth App following the directions [here](https://docs.github.com/en/apps/oauth-apps/building-oauth-apps/creating-an-oauth-app). When creating the OAuth App, it is fine to enter a placeholder value for the Authorization Callback URL field, as this field will need to be updated to hold a value from Keycloak after more setup steps are complete. The Homepage URL field can be any value.
+
+Locate the client ID and client secret values from the Oauth App page, as they will be required to create the custom resource file.
+
+### Log in to Keycloak console
+
+You will need to log in to the Keycloak console to complete some steps of this process. Once [RHTAS is installed](https://github.com/securesign/sigstore-ocp/blob/main/tas-easy-install.sh), you can do this by finding the route and admin password.
+
+```shell
+$ oc get routes -n keycloak-system | awk '/keycloak/ {print $2}' | head -1
+keycloak-keycloak-system.apps.rosa.umebj-fwkt3-8j8.y6tr.p3.openshiftapps.com
+```
+Copy this route, adding `https://` in front, into your browser.
+
+You will need to log in. The username will be `admin`. You can retrieve the password with:
+```shell
+$ oc get secret/credential-keycloak -n keycloak-system -o yaml | awk '/ADMIN_PASSWORD:/ {print $2}' | base64 -d; printf "\n"
+```
+
+### Add Github Identity Provider to Keycloak Realm
+
+Select one of the options below to set up identity federation to the Github Identity Provider. 
+
+#### Option 1: Create a KeycloakRealm in OpenShift on the CLI
+
+Create a Custom Resource file from which to create the KeycloakRealm that includes the Github identity provider:
```

Comment:
Well, they would need to delete it, as far as I'm aware (described below), or create it with a different name.

Reply:
I'm going through a flow where they make one with a new name right now, to test that, but I think it is actually easier if they delete the existing one. I get this error
```
status:
  loginURL: ""
  message: 'failed to create realm: (409) 409 Conflict'
  phase: failing
  ready: false
```
on the new Keycloakrealm when I haven't deleted the pre-existing `sigstore` one - I think because the IDP already exists in a different Keycloakrealm, but I am not sure.